R/download_buttons.R

```diff
@@ -3,6 +3,20 @@
 
 ## Sizing functions for forest plots ##
 
+#' Calculate the pixel height of a node-split forest plot for a given number of comparisons
+#' 
+#' @param ncomp The number of comparisons in the plot
+#' @return The height of the plot in pixels
+NodePixels <- function(ncomp) { 
```

Comment:
Up to you, but I would have done this in one line:

return(max(400, ncomp * 80))